src/examples/atoms/FlashMessage/Example.js

```diff
@@ -0,0 +1,22 @@
+import React, { Fragment } from 'react'
+import FlashMessage from '../../../components/molecules/FlashMessage'
+import Card from '../../../components/atoms/Card'
+import Button from '../../../components/atoms/Button'
+
+const FlashMessageExample = () => {
+  return (
+    <Card style={{ padding: '30px' }}>
+      <FlashMessage show="true" count={3} text="Link alert message goes here">
+        <Button outline white>
```

Comment:
change this to boolean,

like this `<FlashMessage show />`

Reply:
Updated.